versionedRouter.js

```diff
@@ -611,6 +612,41 @@ if (transformerTestModeEnabled) {
       ctx.status = 400;
     }
   });
+  /* *params
+   * code: transfromation code
+   * language
+   * name
+   * testWithPublish: publish version or not
+   */
+  router.post("/transformation/sethandle", async ctx => {
```

Comment:
"/transformation/sethandle" -> too generic

Reply:
intended to be generic